CLanguage/Interpreter/CInterpreter.cs

```diff
@@ -270,14 +270,19 @@ public Value RunFunction (Value functionAddress, Value arg0, Value arg1, Value a
             Push (arg1);
             Push (arg2);
             Call (functionAddress);
-            return StepFunction (microseconds);
+            return StepFunction (microseconds, 3);
         }
 
-        Value StepFunction (int microseconds)
+        Value StepFunction (int microseconds, int argCount = 0)

```

Comment:
I don't like the default here since the wrong value will cause crashes. Also, all call sites are known, so there's no point.